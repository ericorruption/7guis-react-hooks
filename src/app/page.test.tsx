import React from 'react';
import { render } from '@testing-library/react';
import Home from '@/app/page';

test('renders learn react link', () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});
