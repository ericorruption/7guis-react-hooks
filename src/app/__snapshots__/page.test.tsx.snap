// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div>
  <main
    class="min-h-screen p-24"
  >
    <label>
      <input
        type="number"
        value=""
      />
       
      Celsius
    </label>
     
    =
     
    <label>
      <input
        type="number"
        value=""
      />
       
      Fahrenheit
    </label>
  </main>
</div>
`;
